bash/t/terraform/terraform_run.functions

```diff
@@ -72,8 +72,8 @@ t_tf() {
     run_t t_use_which_if_tf_var_not_set
 }
 
-### export_governance_vars
-t_export_governance_vars() {
+### tf_export_governance_vars
+tf_t_export_governance_vars() {
```

Comment:
This name won't work - for the test runner to pick up the test, it should be prefixed t_tf, not tf_t ;)

Reply:
Argghh! It's my stupidity though. I've made multiple selection in the editor and that's the result. Will fix this.